### PR TITLE
Add PFS 4-12 Actors

### PR DESCRIPTION
**Pull request by MiraculousWaterBottle**

### CODE REVIEW

File: packs/data/pfs-season-4-bestiary.db/witchs-eyes.json [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
+                "source": {
+                    "value": ""
+                }
+            },
+            "type": "lore"
+        },
+        {
+            "_id": "UVpSLyWiyEs0CxqR",
+            "img": "systems/pf2e/icons/default-icons/lore.svg",
+            "name": "Owl Coven",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-eyes.json:606]: Should this be "Owl Coven Lore"?

File: packs/data/pfs-season-4-bestiary.db/witchs-gazes.json [hunk 2]
```diff
@@ -0,0 +1,774 @@
...
+                "source": {
+                    "value": ""
+                }
+            },
+            "type": "lore"
+        },
+        {
+            "_id": "UVpSLyWiyEs0CxqR",
+            "img": "systems/pf2e/icons/default-icons/lore.svg",
+            "name": "Owl Coven",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-gazes.json:627]: Should this be "Owl Coven Lore"?

File: packs/data/pfs-season-4-bestiary.db/witchs-talons.json [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+                "source": {
+                    "value": ""
+                }
+            },
+            "type": "lore"
+        },
+        {
+            "_id": "2bEwGqomZ3nJEBGL",
+            "img": "systems/pf2e/icons/default-icons/lore.svg",
+            "name": "Owl coven",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-talons.json:379]: Should this be "Owl Coven Lore"?

File: packs/data/pfs-season-4-bestiary.db/witchs-wings.json [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+                "source": {
+                    "value": ""
+                }
+            },
+            "type": "lore"
+        },
+        {
+            "_id": "3ZRG2DsoUwXzbf7G",
+            "img": "systems/pf2e/icons/default-icons/lore.svg",
+            "name": "Owl coven",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-wings.json:363]: Should this be "Owl Coven Lore"?

File: packs/data/pfs-season-4-bestiary.db/witch-of-the-owl.json [hunk 5]
```diff
@@ -0,0 +1,1415 @@
...
+            },
+            "blurb": "",
+            "creatureType": "",
+            "level": {
+                "value": 1
+            },
+            "privateNotes": "",
+            "publicNotes": "",
+            "source": {
+                "value": ""
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witch-of-the-owl.json:1369]: Needs a source. Probably `"Pathfinder Society Scenario #4-12: Negotiations for the Star Gun"`

File: packs/data/pfs-season-4-bestiary.db/witchs-eyes.json [hunk 6]
```diff
@@ -0,0 +1,753 @@
...
+            },
+            "blurb": "",
+            "creatureType": "",
+            "level": {
+                "value": 0
+            },
+            "privateNotes": "",
+            "publicNotes": "",
+            "source": {
+                "value": ""
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-eyes.json:712]: Needs a source.

File: packs/data/pfs-season-4-bestiary.db/witchs-gazes.json [hunk 7]
```diff
@@ -0,0 +1,774 @@
...
+            },
+            "blurb": "",
+            "creatureType": "",
+            "level": {
+                "value": 2
+            },
+            "privateNotes": "",
+            "publicNotes": "",
+            "source": {
+                "value": ""
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-gazes.json:733]: Needs a source.

File: packs/data/pfs-season-4-bestiary.db/witchs-talons.json [hunk 8]
```diff
@@ -0,0 +1,500 @@
...
+            },
+            "blurb": "",
+            "creatureType": "",
+            "level": {
+                "value": -1
+            },
+            "privateNotes": "",
+            "publicNotes": "",
+            "source": {
+                "value": ""
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-talons.json:459]: Needs a source.

File: packs/data/pfs-season-4-bestiary.db/witchs-wings.json [hunk 9]
```diff
@@ -0,0 +1,507 @@
...
+            },
+            "blurb": "",
+            "creatureType": "",
+            "level": {
+                "value": 1
+            },
+            "privateNotes": "",
+            "publicNotes": "",
+            "source": {
+                "value": ""
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-wings.json:466]: Needs a source.

File: packs/data/pfs-season-4-bestiary.db/witchs-eyes.json (hunk 1 above)
Comment (MiraculousWaterBottle) [reply packs/data/pfs-season-4-bestiary.db/witchs-eyes.json:606]: Yes It should be

File: packs/data/pfs-season-4-bestiary.db/witch-of-the-owl.json (hunk 5 above)
Comment (TikaelSol) [inline packs/data/pfs-season-4-bestiary.db/witch-of-the-owl.json:1369]:
```suggestion
                "value": "Pathfinder Society Scenario #4-12: Negotiations for the Star Gun"
```

File: packs/data/pfs-season-4-bestiary.db/witchs-talons.json [hunk 10]
```diff
@@ -376,7 +376,7 @@
         {
             "_id": "2bEwGqomZ3nJEBGL",
             "img": "systems/pf2e/icons/default-icons/lore.svg",
-            "name": "Owl coven",
+            "name": "Owl coven Lore",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-talons.json:379]: Uppercase coven

File: packs/data/pfs-season-4-bestiary.db/witchs-wings.json [hunk 11]
```diff
@@ -360,7 +360,7 @@
         {
             "_id": "3ZRG2DsoUwXzbf7G",
             "img": "systems/pf2e/icons/default-icons/lore.svg",
-            "name": "Owl coven",
+            "name": "Owl coven Lore",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/witchs-wings.json:363]: Uppercase coven

File: packs/data/pfs-season-4-bestiary.db/witchs-talon.json [hunk 12]
```diff
@@ -0,0 +1,500 @@
...
+                "source": {
+                    "value": ""
+                }
+            },
+            "type": "lore"
+        },
+        {
+            "_id": "2bEwGqomZ3nJEBGL",
+            "img": "systems/pf2e/icons/default-icons/lore.svg",
+            "name": "Owl coven Lore",
```
Comment (TikaelSol) [inline packs/data/pfs-season-4-bestiary.db/witchs-talon.json:379]:
```suggestion
            "name": "Owl Coven Lore",
```

File: packs/data/pfs-season-4-bestiary.db/witchs-wing.json [hunk 13]
```diff
@@ -0,0 +1,507 @@
...
+                "source": {
+                    "value": ""
+                }
+            },
+            "type": "lore"
+        },
+        {
+            "_id": "3ZRG2DsoUwXzbf7G",
+            "img": "systems/pf2e/icons/default-icons/lore.svg",
+            "name": "Owl coven Lore",
```
Comment (TikaelSol) [inline packs/data/pfs-season-4-bestiary.db/witchs-wing.json:363]:
```suggestion
            "name": "Owl Coven Lore",
```

File: packs/data/pfs-season-4-bestiary.db/witchs-eye.json [hunk 14]
```diff
@@ -0,0 +1,756 @@
...
+                "rules": [
+                    {
+                        "damageType": "bleed",
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "target:condition:flat-footed",
+                            {
+                                "or": [
+                                    "item:damage:type:slashing",
+                                    "item:damage:type:piercing",
```
Comment (TikaelSol) [inline packs/data/pfs-season-4-bestiary.db/witchs-eye.json:539]:
```suggestion
                                    "item:damage:type:piercing"
```

File: packs/data/pfs-season-4-bestiary.db/witchs-gaze.json [hunk 15]
```diff
@@ -0,0 +1,762 @@
...
+                        "damageType": "bleed",
+                        "diceNumber": 1,
+                        "dieSize": "d4",
+                        "key": "DamageDice",
+                        "predicate": [
+                            "target:condition:flat-footed",
+                            {
+                                "or": [
+                                    "item:damage:type:slashing",
+                                    "item:damage:type:piercing",
```
Comment (TikaelSol) [inline packs/data/pfs-season-4-bestiary.db/witchs-gaze.json:546]:
```suggestion
                                    "item:damage:type:piercing"
```

**Pull request merged**